app/models/episode.rb

```diff
@@ -21,8 +21,6 @@ class Episode < ApplicationRecord
 
   acts_as_paranoid
 
-  serialize :categories, JSON
```

Comment:
will this cause issues to have current data no longer using JSON for serializing?